replication/static/src/main/java/io/deephaven/replicators/ReplicateColumnStats.java

```diff
@@ -0,0 +1,46 @@
+package io.deephaven.replicators;
+
+import io.deephaven.replication.ReplicatePrimitiveCode;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.List;
+
+import static io.deephaven.replication.ReplicatePrimitiveCode.shortToAllIntegralTypes;
+import static io.deephaven.replication.ReplicationUtils.globalReplacements;
+
+public class ReplicateColumnStats {
```

Comment:
Is this hooked up to gradle replication tasks?

Reply:
No, thanks for the reminder, I'll update gradle.